include/capo/pcm.hpp

```diff
@@ -35,9 +21,11 @@ struct PCM {
 
 	static constexpr std::size_t max_channels_v = 2;
 
-	SampleMeta meta;
+	Metadata meta;
 	std::vector<Sample> samples;
-	utils::Size size{};
+	std::size_t bytes{};
```

Comment:
I don't see anything wrong with keeping a Size object here instead of a std::size_t. It's more specific.

Reply:
It stores the result as a float + unit, which may cause imprecision. It's meant for human friendly output, not really for precise storage.

Reply:
What about having size store bytes as size_t instead of float?

Reply:
Then the original purpose - of representing the value in the nearest thousandth unit - becomes less precise: numbers beyond decimal point are lost. 🤓